numpy/core/multiarray.py

```diff
@@ -893,8 +893,9 @@ def bincount(x, weights=None, minlength=None):
 
     >>> np.bincount(np.arange(5, dtype=float))
     Traceback (most recent call last):
-      File "<stdin>", line 1, in <module>
-    TypeError: array cannot be safely cast to required type
+      ...
+    TypeError: Cannot cast array data from dtype('float64') to dtype('int64')
+    according to the rule 'safe'
```

Comment:
Unfortunately, this error message is platform-dependent

Reply:
We could potentially use `...` here to satisfy doctest on Windows. But if we're happy with the refguide check passing on only on Linux (which is all we need for CI) then this might be be enough.

Reply:
Perhaps we should assert that before running refguide?

Reply:
Then anyone running the refguide tests on windows would fail, is that really what we want? How often do users actually depend on the exact text in an error messge?

Reply:
I wouldn't assert anything, preventing running the checker completely won't be helpful. Instead, I would document that it is only required to fully pass with the config of TravisCI; other Linux systems will likely pass too; other OSes may show some failures.